src/games/seabattle/sound.ts

```diff
@@ -12,56 +12,14 @@ import MissSound5 from './media/miss5.mp3';
 const HIT_SOUNDS = [HitSound1, HitSound2, HitSound3];
 const MISS_SOUNDS = [MissSound1, MissSound2, MissSound3, MissSound4, MissSound5];
 
-// Visible for testing only
-function isSalvoUpdate(gameArgs: IGameArgs, action: any): boolean {
-  if (gameArgs.mode === GameMode.OnlineFriend) {
-    if (action.type !== 'UPDATE') {
-      return false;
-    }
-    const moveLog = action.deltalog.find(
-      (entry: any) => entry.action.type === 'MAKE_MOVE' &&
-        entry.action.payload.type === 'salvo',
-    );
-    if (!moveLog) {
-      return false;
-    }
-    return true;
+export const SeabattleSound = (sound: string) => {
```

Comment:
if this is a function it should start with lower case, and generally have a verb, so rename it to be like playSeabattleSound